src/natcap/invest/crop_production_regression.py

```diff
@@ -505,32 +505,40 @@ def execute(args):
         args['fertilization_rate_table_path'],
         **MODEL_SPEC['args']['fertilization_rate_table_path'])
 
-    crop_lucodes = list(crop_to_landcover_df[_EXPECTED_LUCODE_TABLE_HEADER])
+    lucodes_in_table = list(crop_to_landcover_df[_EXPECTED_LUCODE_TABLE_HEADER])
 
-    unique_lucodes = numpy.array([])
+    unique_lucodes_in_raster = numpy.array([])
     for _, lu_band_data in pygeoprocessing.iterblocks(
             (args['landcover_raster_path'], 1)):
         unique_block = numpy.unique(lu_band_data)
-        unique_lucodes = numpy.unique(numpy.concatenate(
-            (unique_lucodes, unique_block)))
+        unique_lucodes_in_raster = numpy.unique(numpy.concatenate(
+            (unique_lucodes_in_raster, unique_block)))
 
-    missing_lucodes = set(crop_lucodes).difference(
-        set(unique_lucodes))
-    if len(missing_lucodes) > 0:
+    lucodes_missing_from_raster = set(lucodes_in_table).difference(
+        set(unique_lucodes_in_raster))
+    if len(lucodes_missing_from_raster) > 0:
```

Comment:
I know you're just changing the variable names here, but may I suggest shortening this to
```suggestion
    if lucodes_missing_from_raster:
```
since `bool(set()) is False`

Reply:
Ah, good catch, thanks. I'll update this one and all the analogous expressions.